client/src/interfaces/userInterfaces.ts

```diff
@@ -0,0 +1,13 @@
+
+export interface userDataInterface {
+  name: string,
+  middleName?: string,
+  surname:  string,
+  secondSurname?:  string,
+  birthday: string
+  email: string,
+};
+
+export interface userSignUpInterface extends userDataInterface {
+  password: string,
+};
```

Comment:
please check end of line on all files, maybe set Prettier to help you with things like this